charts/super-agent-deployment/templates/_helpers.tpl

```diff
@@ -0,0 +1,72 @@
+{{- /*
+Return the name of the configMap holding the Super Agent's config. Defaults to release's fill name suffiexed with "-config"
+*/ -}}
+{{- define "newrelic-super-agent.config.name" -}}
+{{- .Values.config.name | default (include "newrelic.common.naming.truncateToDNSWithSuffix" ( dict "name" (include "newrelic.common.naming.fullname" .) "suffix" "config" )) -}}
+{{- end -}}
+
+
+{{- /*
+Return the key name of the configMap holding the Super Agent's config. Defaults to "config.yaml"
+*/ -}}
+{{- define "newrelic-super-agent.config.key" -}}
+{{- .Values.config.key | default "config.yaml" -}}
+{{- end -}}
+
+
+{{- /*
+This function simply templates the default configuration for the agent.
+*/ -}}
+{{- define "newrelic-super-agent.config.defaultConfig" -}}
+opamp:
+  endpoint: COMPLETE-ME
+  headers:
+    api-key: COMPLETE-ME
+{{- end -}}
```

Comment:
why are we not putting this as a default of the values.yaml for `newrelic-super-agent.config.content` for now? 

If I am not mistaken we are going always to override that 🤔

Reply:
Fair. For now, it is the best.

Reply:
Done on [#032a7e5](https://github.com/newrelic/helm-charts/pull/1155/commits/032a7e5c51218d096b4842f6f469f89bb3566fd2)